### DISCUSSION TITLE
Help Wanted: Why don't my panels layout side by side in elk graph view?

### DISCUSSION THREAD

**Discussion by ndtreviv** [f5/unovis#242] [Q&A]
I'm creating a Graph view showing in `elk` layout format. I've copied the example so far, and then changed the data to be my own. For some reason my two sub groups don't layout side by side, instead they layout on top of each other.

Why is this? What have I missed?

<img width="255" alt="Screenshot 2023-06-27 at 15 30 36" src="https://github.com/f5/unovis/assets/1530653/1c79fab8-8aed-443b-9383-61faddbc738d">

**Comment by rokotyan** [comment]
It's difficult to determine the exact cause of the issue without seeing the code or the specific configuration you have used for your graph.

Configuring `elk` can be challenging. Luckily they have an online editor allowing to quickly prototype graph configurations: https://rtsys.informatik.uni-kiel.de/elklive/json.html. And after you get the layout you like, you can copy its configuration to Unovis. 
<img width="838" alt="image" src="https://github.com/f5/unovis/assets/755708/3a8d5ffc-58e2-4760-b27d-adf6903d9f57">

**Reply by ndtreviv** [reply]
I think what I'm struggling with is how do I convert that elk config to something I can use in unovis?

**Reply by rokotyan** [reply]
@ndtreviv Feel free to share your ELK config, we'll try to help
